examples/reference/chat/ChatEntry.ipynb

```diff
@@ -9,7 +9,7 @@
     "import asyncio\n",
     "\n",
```

Comment:
I think you need to move the Chat components to their own chat components section in the Component Gallery too?

Reply:
I'm a bit confused; these Jupyter notebooks are the component gallery items, right? Once the docs are built, it'll create its own section automatically?

Reply:
Ok. I did not know. I thought they where defined in some TocTree somewhere.

Reply:
I'm not 100% sure either; leaving this unresolved for Philipp to see.

Reply:
You need to add a new section in the `nbsite_gallery_conf` definition of `doc/conf.py`

Reply:
Okay added